tests/test_configs.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+
+from piker import config
+
+
+def test_root_conf_networking_section():
```

Comment:
Ok so obviously we need to change the name of this 😂 

I would change it to something like `test_brokersconfig_created_created_from_template_on_load_accounts()`

or something..

Reply:
I'd also add a doc string explaining that this issue was found when trying to run `piker chart` on windows.

some resources to help you:
- https://peps.python.org/pep-0257/#what-is-a-docstring
- https://peps.python.org/pep-0008/#documentation-strings

Also, you can look at other tests which have them to get a rough idea.